test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -861,6 +876,77 @@ protected void createTableMayThrow(String query) throws Throwable
         QueryProcessor.executeOnceInternal(fullQuery);
     }
 
+    protected String createView(String query)
+    {
+        return createView(null, query);
+    }
+
+    protected String createView(String viewName, String query)
+    {
+        String currentView = viewName == null ? createViewName() : viewName;
+        String fullQuery = String.format(query, KEYSPACE + "." + currentView, KEYSPACE + "." + currentTable());
+        logger.info(fullQuery);
+        schemaChange(fullQuery);
```

Comment:
I wonder if we should call here `waitForViewBuild()` bc I can see this being a future source of flakiness...

Reply:
Good idea, that could avoid future mistakes and makes the current tests a bit more concise, done [here](https://github.com/apache/cassandra/pull/1306/commits/464ca8473f5301a14e2f7f57a038e490c84ac7b6).

There are a couple of tests in `ViewTest` (`testViewBuilderResume` and `testTruncateWhileBuilding`) that rely on `createView` not waiting for the build tasks. For those special cases I have added a `CQLTester#createViewAsync` method that keeps the previous behaviour.